src/controllers/landing.controller.ts

```diff
@@ -2,14 +2,13 @@ import { NextFunction, Request, Response } from "express";
 
 import { logger } from "../utils/logger";
 import * as config from "../config";
-import { deleteApplicationData } from "../utils/application.data";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
-    logger.debugRequest(req, `GET LANDING_PAGE`);
+    const landingUrl = config.LANDING_PAGE_URL;
+    logger.debugRequest(req, `GET LANDING_PAGE ${landingUrl}`);
 
-    deleteApplicationData(req.session);
-    return res.render(config.LANDING_PAGE);
+    return res.redirect(landingUrl);
```

Comment:
does the redirect work when we use the live LANDING_PAGE_URL=https://www.gov.uk/guidance/register-an-overseas-entity
not sure if we can redirect outside of our app

Reply:
seems ok, just tried hardcoding the url into my controller in docker and it redirects

Reply:
Tested this and it does go to the gov url.

Reply:
This way we need to add the `?start=0` [here](https://github.com/companieshouse/overseas-entities-web/blob/a65b5d5b4047a143c85abb19b33d1cd8a555cc3f/views/layout.html#L58)

Reply:
The problem that how you going to differentiate between gov and ch page

Reply:
The part related to the link header will be covered by ROE-1082 not in this story.